specificatie/openapi.yaml

```diff
@@ -1005,7 +1016,8 @@ components:
     Nationaliteit:
       type: "object"
       description: |
-                    * **redenOpname** : De reden op grond waarvan de persoon de nationaliteit gekregen heeft.
+                    * **redenOpname** : De reden op grond waarvan de persoon de nationaliteit gekregen heeft. Wordt gevuld op bassi van de waarden die voorkomen in de tabel 'redenopnemenbeeindigennationaliteit' uit de Haal-Centraal-BRP-tabellen-bevragen API.
```

Comment:
Gezien de opmerking van @fsamwel ('De huidige landelijke tabellen hebben een code met onderscores') vraag ik me af of de tabelnaam 'redenopnemenbeeindigennationaliteit' wel de juiste naam is.